File: IntroJava/src/GuessNumber.java
/*
Listing 5.3 GuessNumber.java
 */

import java.util.Scanner;

public class GuessNumber
{
    public static void main(String[] args)
    {
        int randomNumber = (int) (Math.random() * 101);

        Scanner input = new Scanner(System.in);

        int guessNumber = -1;
        while (guessNumber != randomNumber)
        {
            System.out.print("Enter your guess number: ");
            guessNumber = input.nextInt();

            if (guessNumber == randomNumber)
                System.out.println("You are right!");
            else if (guessNumber < randomNumber)
                System.out.println("Your guess is low!\n");
            else
                System.out.println("Your guess is high!\n");
        }


    }
}

File: IntroPython/GuessNumber.py
# LISTING 5.3 GuessNumber.py

import random

number = random.randint(0, 100)
print("Guess a magic number between 0 and 100.")

guess = -1
while guess != number:
    guess = eval(input("Enter your guess: "))
    if guess == number:
        print("You are right!")
    elif guess > number:
        print("Your guess is too high!")
    else:
        print("Your guess is too low!")
